pycondor/cli.py

```diff
@@ -196,9 +196,12 @@ def monitor(time_, length, prog_char, file):
                                         prog_char=prog_char)
             sys.stdout.write(prog_str)
             sys.stdout.flush()
-            # Exit if all jobs are either Done or Failed
-            n_finished = current_status.Done + current_status.Failed
-            if n_finished == sum(current_status) and sum(current_status) != 0:
+            # Exit if at least one job is Done or Failed (something ran) and
```

Comment:
@duncanmmacleod I'm unsure if this comment matches what the logic implements. The first sentence implies it will exist if any job as Done or Failed, but the logic (as I understand it) is that it will exit if all jobs are either done, failed, or their parents failed. Have I misunderstood or does the comment need rephrasing?

Reply:
Okay, go it. I'm happy with it as is. I was misunderstanding previously.